### PR TITLE
Initial QNN module

### PR DESCRIPTION
**Pull request by Chriun**
Somewhat working QNN (training plateauing, need to find optimal initial weights/learning rate/optimizer). Need to format as a module (step-by-step walkthrough) and test on IBM system.

### CODE REVIEW

**Comment by Chriun** [comment]
Note: You can view jupyter notebooks and diffs between old vs new in Github Feature Preview. Click on your profile on the top right and click on Feature Preview. Then click on Rich Jupyter Notebook Diffs and enable it.

https://github.blog/changelog/2023-03-01-feature-preview-rich-jupyter-notebook-diffs/
